umbraco/Umbraco-CMS
# v9: Update cypress to newest version and fix errors

# Notes
- Updated cypress to 8.6.0
- Fixed Scripts & Stylesheets, we were going to fast so worker was throwing network errors.

# How to test
- Run a clean install of umbraco
- Go to Umbraco.Tests.AcceptanceTests and run `npm install` & `npm run test`
- All tests should pass
